extending-valtimo/server-sent-events.md

```diff
@@ -0,0 +1,75 @@
+# Server sent events
+
+Server sent events is a paradigm used to send information from the server to the client
+without an explicit request. 
+
+As such Valtimo provides an implementation of Server sent events to allows
+web clients to listen to specific events or for implementations to create their
+own events to be sent
+
+### Creating events
+
+To create events implementations only need to extend the BaseSseEvent class
+and add any number of properties.
+
+```kotlin
+abstract class BaseSseEvent(val eventType: String) //provided by Valtimo
+
+class CampaignSseEvent(
+        val offerId: UUID,
+        val offerDescription: String
+    ) : BaseSseEvent("NEW_OFFER_EVENT") //example implementation
+```
+
+### Sending events
+
+Valtimo provides a Spring Bean as a service to allow for events to be sent to
+subscribers.
+
+To do so Valtimo the SseSubscriptionService bean can be used as in the example bellow.
```

Comment:
This sentence doesn't make sense as-is

Reply:
what do you mean?

Reply:
> While there's still some details to be discussed about security, maybe you could add that no sensitive data should be transferred using these events.

@theo-ritense i've added a commit with a disclaimer to address this. please review